### ISSUE TITLE
Receiving parallel to sending frames

### ISSUE THREAD

**Issue by Alick09** [aiortc/aiortc#332]
I am trying to modify server example to make receiving and sending frames parallel.

The problem appeared when I've modified frame processing and it became too long. So I need to find out the way to deal with it. Now I see 2 ways to do it.
1. Understand input frame queue length. So I can read more when queue is too big.
2. Read frames as fast as it comes in parallel process.

For the first way I don't understand how to get queue length. Is it possible for WebRTC at all? Maybe this way is wrong.
For the second way I've tried to run in parallel process but in parallel receiving frames not worked in my implementation. believe it's more possible than first way but can't do it for now.

**Comment by jlaine** [comment]
The question doesn't really seem to be specific to `aiortc`, nor about "sending and receiving in parallel". If I understand your question correctly it seems to be more along the lines "how do I do multi-threaded / multi-process" processing?

**Comment by Alick09** [comment]
The problem is as I undersood that it's very hard (or even impossible) to run daemon thread inside asyncio loop which will call one of the async methods. It would be very helpful if you know how to do it and you will explain.

**Comment by jlaine** [comment]
I'm going to close this as out of scope, I can't provide asyncio tutorials.
